src/test/java/org/cactoos/scalar/RetryScalarTest.java

```diff
@@ -44,8 +44,9 @@
 
     @Test
     public void runsScalarMultipleTimes() throws Exception {
-        MatcherAssert.assertThat(
-            new RetryScalar<>(
+        new Assertion<>(
+            "Should run until success",
+            () -> new RetryScalar<>(
                 () -> {
```

Comment:
In all of the tests you forget to call Assertion<>().affirm(). Without that tests are not executed

Reply:
good catch, thanks!